src/Microsoft.ML.EntryPoints/ImportTextData.cs

```diff
@@ -3,17 +3,17 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.ML.Data;
-using Microsoft.ML.Runtime;
-using Microsoft.ML.Runtime.CommandLine;
-using Microsoft.ML.Runtime.Data;
-using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML;
+using Microsoft.ML.CommandLine;
+using Microsoft.ML.EntryPoints;
+using System;
```

Comment:
Know why System got added?

Reply:
Probably due to a dependency. The next PR on this effort will sort the namespaces and remove unused namespaces from the entire codebase so I wouldn’t worry about potential unused namespace.